src/lib/layout/footer.svelte

```diff
@@ -79,7 +79,18 @@
         margin-block-start: auto;
     }
 
+    .main-footer-start .inline-links-item {
+        padding-inline-end: 1rem;
+        padding-inline-start: unset;
+    }
+
     [class^='icon-']:not(.icon-cloud):not(:hover) {
         color: hsl(var(--color-neutral-50));
     }
+
+    @media (max-width: 767.99px), (min-width: 768px) and (max-width: 1198.99px) {
```

Comment:
Why not `@media (max-width: 1198.99px)`? And isn't 1198.99 oddly specific?

Reply:
I don't remember exactly why this was this specific but I am pretty sure it might've broke on 1199 ig. I'll re-check.

Reply:
Done.